docs/index.rst

```diff
@@ -33,17 +47,16 @@ Install Haiku by running::
 
     $ pip install git+https://github.com/deepmind/dm-haiku
 
-.. toctree::
-   :caption: Guides
-   :maxdepth: 1
 
-   modules
 
-.. toctree::
-   :caption: Package Reference
-   :maxdepth: 1
+Known issues
```

Comment:
Consider including a link from README to here, under the new heading "known issues"?
We can do this later too, if finding the section-hyperlink is too much trouble.

Reply:
Okay, let's do it in a follow up commit.